src/xmake.lua

```diff
@@ -27,6 +27,15 @@ target("libco")
         add_packages("openssl")
     end 
 
+    if is_kind("shared") then
+        set_symbols("hidden")

```

Comment:
I have no experience with xmake, but it seems that this abstraction set è-fvisibility=hidden` in GNU like compilers.

Reply:
> I have no experience with xmake, but it seems that this abstraction set `-fvisibility=hidden` in GNU like compilers. I don't know whether it also injects `-fvisibility-inlines-hidden`.

@waruqi may help with this.

Reply:
It will also add inline hidden.

https://github.com/xmake-io/xmake/blob/3a131d8e0b51115bae73896fd6e85e34452801f8/xmake/modules/core/tools/gcc.lua#L104